common-service-core/src/main/java/com/ca/mfaas/message/core/AbstractMessageService.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.mfaas.message.core;
+
+import com.ca.mfaas.message.storage.MessageTemplateStorage;
+import com.ca.mfaas.message.template.MessageTemplate;
+import com.ca.mfaas.message.template.MessageTemplates;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Abstract class which implements the {@link MessageService} interface.
+ * It creates messages from the file, validate and add them to the {@link MessageTemplateStorage}.
+ */
+@Slf4j
+public abstract class AbstractMessageService implements MessageService {
+
+    private final MessageTemplateStorage messageTemplateStorage;
+
+
+    /**
+     * Constructor that creates common messages and messages from file.
```

Comment:
"... the file" or "... a file" depending on whether users know what file you referring to. IF specific - use "the", if generic - "a". In the given context, it seems you users should know what specific file is being mentioned.

Reply:
`Constructor that creates by default messages from the common file and also from a specified file. `
Do you think it makes sense?

Reply:
"Constructor that creates messages from the common file as well as from a specified file." 
2 points:
1 - what is 'the common file'?
2 - I think, we can omit 'by default' in this sentence. If you say we need it there, I'd rather say "Constructor that automatically (or 'defaultly') creates..." Let me know if this 'by default' is necessary to mention.